examples/mood.py

```diff
@@ -0,0 +1,43 @@
+"""MooD - a pacifist/tourist meta-mod for Doom
+
+All monsters are pacifist and will not attack the player or each other.
+The pistol has been replaced by the fist.
+All weapon pickups have been replaced by the Health Potion.
+
+For more information: https://jmtd.net/doom/mood/"""
+
+from deh9000 import *
+
+# pacifist monsters
+for monster in mobjinfo:
+    if not (monster.flags & MF_COUNTKILL or monster == mobjinfo[MT_SKULL]):
```

Comment:
Nit: 

    if not (monster.flags & MF_COUNTKILL) and monster != mobjinfo[MT_SKULL]: